src/main/java/org/jabref/gui/entryeditor/OtherFieldsTab.java

```diff
@@ -35,7 +35,7 @@
 public class OtherFieldsTab extends FieldsEditorTab {
 
     public static final String NAME = "Other fields";
-    private final List<Field> customTabFieldNames;
+    private final Set<Field> customTabFieldNames;
```

Comment:
This neeeds to be a `List`, because the user determines the order in which the tab appears. We should not depend in `LinkedHashSet` to enforce an order. `List` is the much more cleaner way!

Reply:
That is an artifact from something i experimented with and decided not to put effort in.
But actually in this case the Set collects the fields, not the tabs. But i will undo the change anyway.